warp10/src/main/java/io/warp10/continuum/Configuration.java

```diff
@@ -109,6 +109,11 @@
    */
   public static final String CONFIG_WARPSCRIPT_TIMEBOX_MAXTIME = "warpscript.timebox.maxtime";
 
+  /**
+   * Capability name used for extending timebox maxtime, if set.
+   */
+  public static final String MAXTIME_EXTENSION_ALLOWANCE_CAPNAME = "timebox.maxtime.extension.capname";
```

Comment:
Change configuration key name to make it coherent with `warpscript.timebox.maxtime`. Use `warpscript.timebox.maxtime.capname` for instance.

Reply:
Also add it, commented, to the configuration templates to make it more visible.